### PR TITLE
Created ModelCard for Hel-ach-en MT model

### PR DESCRIPTION
**Pull request by Pogayo**
# What does this PR do?

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to the it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors which may be interested in your PR.

### CODE REVIEW

File: model_cards/Ogayo/Hel-ach-en/README.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+language: 
+- ach 
+- en
+tags:
+- translation
+- Pytorch
```
Comment (julien-c) [inline model_cards/Ogayo/Hel-ach-en/README.md:7]:
```suggestion
```

unneeded

File: model_cards/Ogayo/Hel-ach-en/README.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+license: CC By 4.0
```
Comment (julien-c) [inline model_cards/Ogayo/Hel-ach-en/README.md:8]:
```suggestion
license: cc-by-4.0
```
we use the identifiers from https://docs.github.com/en/free-pro-team@latest/github/creating-cloning-and-archiving-repositories/licensing-a-repository#searching-github-by-license-type

**Comment by julien-c** [comment]
This is really cool @Pogayo, thanks for sharing.

If you can, please consider adding sample inputs for the inference widget, either in DefaultWidget.ts (see https://huggingface.co/docs#how-can-i-control-my-models-widgets-example-inputs) or in this model card.

Will also add Acholi to the list in https://huggingface.co/languages

**Comment by Pogayo** [comment]
I don't know if this is the right place to ask, apologies in advance - I am trying to translate on the model page and getting this error:
image
I have not been able to figure out what causes it so if you can guide me, I would really love to see this model accessible for people.

Unrecognized configuration class for this kind of AutoModel: AutoModelForCausalLM. Model type should be one of CamembertConfig, XLMRobertaConfig, RobertaConfig, BertConfig, OpenAIGPTConfig, GPT2Config, TransfoXLConfig, XLNetConfig, XLMConfig, CTRLConfig, ReformerConfig, BertGenerationConfig, XLMProphetNetConfig, ProphetNetConfig.

**Comment by Narsil** [comment]
Did you change the `pipeline_tag` in the meantime ? It's working now:

https://huggingface.co/Helsinki-NLP/opus-mt-luo-en?text=Ariyo

The error seemed to point it wanted to do text-generation with your model which it can't.

**Comment by Pogayo** [comment]
It is still not working @Narsil. I get a different error now, do you know what might be causing it?
image

The model you referenced is a different one- A Luo -English model- This one is Acholi -English

**Pull request merged**